### DISCUSSION TITLE
What is the replacement for `app.renderer.plugins.interaction.on` in v7?

### DISCUSSION THREAD

**Discussion by Chaos-Enyo** [pixijs/pixijs#9422] [Q&A] [answered]
What is the replacement for `app.renderer.plugins.interaction.on` in v7?

**Comment by bigtimebuddy** [comment, marked as answer]
If you want to add global event listeners, consider adding event listeners to your root container `app.stage.on`. We also added the following:

* `globalmousemove`
* `globalpointermove`
* `globaltouchmove`

Events now support bubbling, so you should be able to use the return `event.target` to get the element you interacted with.

**Reply by Chaos-Enyo** [reply]
Thank you!

**Reply by whitespacecode** [reply]
@bigtimebuddy 
Could you update the docs for globalpointermove?
I was searching for crazy why this didn't work on my code and the reason was eventFeatures on Application. globalMove was set to false
```
 eventFeatures: {
            move: true,
            globalMove: true, //set to true if you want to use globalpointermove
            click: true,
            wheel: false,
        }
```

**Reply by bigtimebuddy** [reply]
PR welcome to clarify that the global move events.
